AEM/getting-started-with-automixedprecisionmkl.md

```diff
@@ -294,7 +268,7 @@ To enable them in other operations, rebuild TensorFlow with the appropriate comp
 ```
 Again, notice 2 operators are converted into BFloat16 type. The script also supports dumping the model into protobuf binary file (.pb) or text file (.pbtxt).
 
-### Controlling the Operators that will be Ported to BFloat16 type Automatically
+## Appendix 2 : Controlling Ops that will be converted to BFloat16 type Automatically
```

Comment:
Instead of keeping this in an Appendix, move under the legacy section (as this working only with legacy path)

Reply:
ok, will move it back under Legacy. Legacy section was getting too big so I had removed it from there and moved it here. But sure, I will move it back